index.html

```diff
@@ -15,20 +17,29 @@
 
```

Comment:
[ **OPTIONAL** ] Great Job @jmonto55. Your project looks promising. Hurray! 🥳 🥳 🥳 There is one suggestion though, it seems like the image `src` is missing which is making the logo return a 404 error. It would make your website complete if you could fix the logo 🙂

Reply:
Thank you for the advice @RiyaBulia12 I will fix the logo right now before merging :)